### PR TITLE
Added view to show location summary via the mode GET parameter

### LINKED ISSUE #1: Add location display mode to new Plate Tectonics model
This will display information about the specified location in place of the full model (e.g. prediction boundary type + explanation text).
- Specify with URL params?
- Show location info (name, images?) and student's latest prediction (attempt) data
- Show prediction for each location in this step: https://wise.berkeley.edu/preview/unit/33873/node264

Model repo: https://github.com/davidchuyaya/plate-tectonics

Deadline: Feb 3, 2021 (runs start Feb 8)

### PR DESCRIPTION
**Pull request by hirokiterashima**
**Description**
I added a location summary view for this model that can be shown by passing in a mode=showLocationSummary GET parameter. The location can be specified by an additional location=[ccd/ooc/ood/cod/coc] GET param. Right now the view only shows what you typed in the prediction textarea.

**Testing**
1. Create an embedded component and set its url to 
```
pt-boundary-explorer/index.html
```

2. In another step, create another embedded component, connect it to the first PT model component using "connected components", and set its url to
```
pt-boundary-explorer/index.html?mode=showLocationSummary&location=ccd
```

3. As a student, do work on location ccd in the first embedded component. Remember what you wrote in the prediction textarea.
4. Go to the second embedded component. It should display what you wrote in the previous step.

Closes #1

### CODE REVIEW

**Review by geoffreykwan — APPROVED** [review]
Seems to work well. I think the locations should be ooc/coc/ood/ccd/ccc. I don't think there is a cod location.

**Comment by breity** [comment]
Hey @geoffreykwan and @hirokiterashima, I styled and updated with options to display location text, boundary type selections, and student text. These can be set by URL params as well (`showLocationText`, `showBoundarySelection`, `showStudentText`) and all default to true.

Can you take another look? You can see it working here: https://wise.berkeley.edu/preview/unit/33873/node264.

**Comment by hirokiterashima** [comment]
@breity Looks great! The location summary display message is easy to read and digest. 👍  I had one comment about the code, see below.

File: src/App.js [hunk 1]
```diff
@@ -212,22 +214,46 @@ function App() {
...
+    if (attempts) {
+      locationAttempts = attempts.filter((attempt) => {
+        return attempt.selectedExample === location;
+      });
+    }
+    return locationAttempts;
+  }
+
+  function showLocationAttempt(lastAttempt, location) {
+    const showLocationText = getParam('showLocationText') === 'false' ? false : true;
```
Comment (hirokiterashima) [inline src/App.js]: This tertiary check is a bit hard to read. How about this?
```
const showLocationText = getParam('showLocationText') !== 'false';
```

**Review by breity — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (breity) [reply src/App.js]: Good suggestion, thanks. I updated.

**Review by geoffreykwan — APPROVED** [review]
Looks good.

**Pull request merged**